README.md

```diff
@@ -31,15 +31,15 @@ These endpoints no longer return HATEOAS links:
 - `Delete an Historic Non FHL UK Property Annual Submission`
 
 #### self-assessment-accounts-api v3
-- Added optional `poaRelevantAmount` to `Retrieve Self Assessment Balance and Transactions` for tax years from 2024-25.
+- Added optional `poaRelevantAmount` to `Retrieve Self Assessment Balance and Transactions`.
```

Comment:
Is this parameter supposed to be visible right now in https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-accounts-api/3.0/oas/page#tag/Payments-and-Liabilities/paths/~1accounts~1self-assessment~1%7Bnino%7D~1balance-and-transactions/get ?

It's a good idea to clarify what `poaRelevantAmount` is and where it is located.

Are you saying the following?
Added optional parameter `poaRelevantAmount` to response body of endpoint `Retrieve Self Assessment Balance and Transactions`